pkg/endpoint/redirect_test.go

```diff
@@ -19,23 +19,21 @@ package endpoint
 import (
 	"context"
 
+	"gopkg.in/check.v1"
```

Comment:
Minor nit: this import should go after the `github.com/cilium/cilium` imports into a separate group. We usually use the order stdlib, cilium, 3rd party for imports.

Reply:
@tklauser Once the PR is ready to merge, I can fix the imports. Else, I'm anyway planning to do a follow-up so I fix it then.